shallow_backup/printing.py

```diff
@@ -37,6 +37,22 @@ def print_green_bold(text):
 	print(Fore.GREEN + Style.BRIGHT + text + Style.RESET_ALL)
 
 
+def print_blue_bold_path(text, path):
+	print(Fore.BLUE + Style.BRIGHT + text, Style.NORMAL + path + Style.RESET_ALL)
```

Comment:
Should there be a space between the message and the path?

Reply:
Yep, so the format for this will always look like this.

```
MESSAGE HERE: PATH
```

It's fine to hardcode the space in there.